CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## Fleet 4.44.1 (Feb 12, 2024)
+
+### Bug fixes
+
+* Fix a bug where long enroll enroll secrets would overlap with the action buttons on top of them.
+* Fixed a bug that would cause OS Settings to never get verified if the MySQL config of Fleet's database has `only_full_group_by` mode enabled (enabled by default).
+* Fix visual display of chevron right icons on Chrome
+* Renamed the `mdm_windows_configuration_profiles` and `mdm_apple_configuration_profiles` `updated_at` field to `uploaded_at` and removed the automatic setting of the value, set explicity instead.
+* Fix a small alignment bug in the setup flow
+* Improved the validation of Windows profiles to prevent errors when the
+  profiles are delivered to the hosts. If you need to embed a nested XML
+  structure (for example for Wi-Fi profiles) you can either:
+      - Escape the XML
+      - Use a wrapping `<![CDATA[ ... ]]>` element
+* Fixes issue where an inaccurate message was returned after running an async (queued) script.
+* Fix URL query params to reset when switching tabs
+* Fix vulnerable software dropdown from switching back to all teams
+* Updated `GET /api/v1/fleet/hosts/:id/activities/upcoming` response to include the count of all
+  upcoming activities for the host.
+* Fixes issue where software from a Parallels VM on a MacOS host would show up in Fleet as if it
+  were the host's software.
+* Removed unnecessary nested database transactions in batch-setting of MDM profiles.
+* Added count of upcoming activities to host vitals UI.
```

Comment:
Please update all change entries so that they all: 

1. Use consistent punctuation and capitalization (periods at the end of all sentences, sentence casing).
2. Past tense ("fixed" not "fix"). 

I use ChatGPT for this. I paste all the changes in with these instructions: 

```
I am creating a changelog for an open source project from a list of commit messages. Please format it for me using the following rules: 

1. Correct spelling and punctuation. 
2. Sentence casing. 
3. Past tense. 
5. Each list item is designated with an asterisk. 
6. Output in markdown format within a code block so I can copy and paste. 
```

Reply:
I ran it through ChatGPT, here's the result. What'd be _really_ cool is if we automatically handled this as part of a "Prepare release" script. 

```
* Fixed a bug where long enroll secrets would overlap with the action buttons on top of them.
* Fixed a bug that caused OS Settings to never get verified if the MySQL config of Fleet's database had `only_full_group_by` mode enabled (enabled by default).
* Fixed visual display of chevron right icons on Chrome.
* Renamed the `mdm_windows_configuration_profiles` and `mdm_apple_configuration_profiles` `updated_at` field to `uploaded_at` and removed the automatic setting of the value, set explicitly instead.
* Fixed a small alignment bug in the setup flow.
* Improved the validation of Windows profiles to prevent errors when the profiles were delivered to the hosts. If you needed to embed a nested XML structure (for example, for Wi-Fi profiles), you could either:
  - Escape the XML
  - Use a wrapping `<![CDATA[ ... ]]>` element.
* Fixed an issue where an inaccurate message was returned after running an async (queued) script.
* Fixed URL query params to reset when switching tabs.
* Fixed vulnerable software dropdown from switching back to all teams.
* Updated `GET /api/v1/fleet/hosts/:id/activities/upcoming` response to include the count of all upcoming activities for the host.
* Fixed an issue where software from a Parallels VM on a MacOS host would show up in Fleet as if it were the host's software.
* Removed unnecessary nested database transactions in batch-setting of MDM profiles.
* Added count of upcoming activities to host vitals UI.
```

Reply:
Haven't used chatGPT outside of the ui so this would be a nifty thing to start trying the api